### ISSUE TITLE
[improvement] let the icon attribute also can use "Remotely loaded" in the future

### ISSUE THREAD

**Issue by bau720123** [phonegap/phonegap-plugin-push#301]
according this，we can use the Remotely loaded to set the image attribute
```
{
    title:"Large Icon",
    message: "Loaded from URL",
    image: "https://XXX/CCC.png"
}
```

well...
but according to this
https://github.com/phonegap/phonegap-plugin-push#action-buttons
I also try let the icon attribute Remotely loaded，but not work，still empty
![screenshot_2015-11-04-14-22-00](https://cloud.githubusercontent.com/assets/5110085/10931365/b516319c-8301-11e5-8599-dda460dda5bc.jpg)

so...just ask
is it possible to let the icon attribute also can use "Remotely loaded" in the future

**Comment by macdonst** [comment]
@bau720123 yeah, right now there is no way to specify the action button icon and sound name remotely.

**Comment by bau720123** [comment]
thanks @macdonst reply
well...another question
according to this
https://github.com/phonegap/phonegap-plugin-push#action-buttons
it told us that the img should put on in the res/drawable folder
for example "emailGuests" and "snooze"
are they png format ?

**Comment by macdonst** [comment]
@bau720123 yes, it is best to use PNG format and when you send the notification from GCM you don't need to add the extension as Android is smart enough to find it in the res/drawables folder.

**Comment by bau720123** [comment]
got it
thanks @macdonst reply

**Comment by bau720123** [comment]
hi @macdonst 
just a feedback
according to this，this is a themebrowser 3rd plugin
https://github.com/initialxy/cordova-plugin-themeablebrowser#experimental-properties
it can set the "wwwImage" attribute
this attribute is very important for PGB's user
because PGB's user can't put images in res/drawable folder by ourself

```
wwwImage is like image but loads image from Cordova's www directory instead. This is a fallback solution when you cannot import native resources. Use image property as much as possible.
```
so...if "phonegap-plugin-push" have this feature，I think it's more better

**Comment by macdonst** [comment]
@bau720123 the problem is when you are adding an action to the push notification the method is:

`addAction (int icon, CharSequence title, PendingIntent intent)`

where `int icon` is an identifier in the resources folder. I'd love to be able to use a remote image but it doesn't look like it is possible.

See: http://developer.android.com/intl/pt-br/reference/android/support/v4/app/NotificationCompat.Builder.html#addAction(int,%20java.lang.CharSequence,%20android.app.PendingIntent)

**Comment by bau720123** [comment]
oh...thanks @macdonst

**Comment by nicoabie** [comment]
Hi @macdonst, Could we send the icon's image name, look for its id and show it?

**Comment by macdonst** [comment]
@nicoabie that's what it does right now. You send the icon's name and it tries to find it in Android drawables.

**Comment by nicoabie** [comment]
@macdonst absolutely right, my bad sorry.
